pkg/workspace/plugins.go

```diff
@@ -351,14 +351,14 @@ func HasPluginGTE(plug PluginInfo) (bool, error) {
 }
 
 // GetPolicyDir returns the directory in which policies on the current machine are managed.
-func GetPolicyDir() (string, error) {
-	return GetPulumiPath(PolicyDir)
+func GetPolicyDir(orgName string) (string, error) {
+	return GetPulumiPath(PolicyDir, orgName)
 }
 
 // GetPolicyPath finds a PolicyPack by its name version, as well as a bool marked true if the path
```

Comment:
Sorry for the late comment, but do you think the comment should be updated now that this func requires the org name as well?

Reply:
Address this in another PR I have open in this repo. Added you as a reviewer there!